care/facility/summarisation/tests_summary.py

```diff
@@ -0,0 +1,104 @@
+from care.facility.models import FacilityRelatedSummary, Facility, PatientSample
+
+from celery.decorators import periodic_task
+from celery.schedules import crontab
+
+from django.core.exceptions import ObjectDoesNotExist
+from django.conf import settings
+from django.utils import timezone
+
+from rest_framework.generics import get_object_or_404
+from rest_framework.mixins import ListModelMixin, RetrieveModelMixin
+from rest_framework.permissions import IsAuthenticated
+from rest_framework import serializers
```

Comment:
Codacy found an issue: [third party import "from rest_framework import serializers" should be placed before "from care.facility.models import FacilityRelatedSummary, Facility, PatientSample"](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5838242)